main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: grandfatherClock
+//  action 1: the grandfather clock ticks
+grandfatherClock.tick();
+//  action 2: the grandfather clock tocks
+grandfatherClock.tock();
+//  action 3: the grandfather clock chimes
+grandfatherClock.chime();
+//  2) 
+//  Noun: baby
+//  action 1: the baby cries
+baby.cry();
+//  action 2: the baby sleeps
+baby.sleep();
+//  action 3: the baby laughs
+baby.laugh();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.